internal/pkg/template/templates/environment/partials/cdn-resources.yml

```diff
@@ -44,9 +44,23 @@ UniqueAliasesAction:
   Properties:
     ServiceToken: !GetAtt UniqueJSONValuesFunction.Arn
     Aliases: !Ref Aliases
+    {{- if .CDNConfig.Static}}
+    AdditionalAlias: {{.CDNConfig.Static.Alias}}
+    {{- end}}
     FilterFor: !Ref ALBWorkloads
 {{- end}}{{/* endif or .CDNConfig.ImportedCertificate .DelegateDNS */}}
 
+{{- if .CDNConfig.Static}}
+CloudFrontOriginAccessControl:
+  Type: AWS::CloudFront::OriginAccessControl
```

Comment:
nit: should we include a 
```
Metadata:
    'aws:copilot:description': 'Access control to make the content in the S3 bucket only accessible through CloudFront
```

Reply:
Hmm I thought about this but then feel like we probably don't want to show this to the customer since they don't need to refer/use/care about this resource at all.

Reply:
Oh that makes sense i'll add it